# adjust test tolerances
--- testsuite/python/lb_circular_couette.py
+++ testsuite/python/lb_circular_couette.py
@@ -126,7 +126,7 @@ class LBCouetteTest:
 
         # check velocity is zero for the radial and axial components
         # (the absolute tolerance follows a Poisson distribution)
-        np.testing.assert_allclose(v_r, 0., atol=1e-4)
+        np.testing.assert_allclose(v_r, 0., atol=2e-4)
         np.testing.assert_allclose(v_z, 0., atol=4e-6)
 
         # check azimuthal velocity is zero inside boundary
@@ -143,7 +143,7 @@ class LBCouetteTest:
         a_ref, b_ref = taylor_couette(slip_vel, 0.0, cyl1.radius, cyl2.radius)
         v_phi_ref = a_ref * r + b_ref / r
         v_phi_drift = np.mean(v_phi) - np.mean(v_phi_ref)
-        np.testing.assert_allclose(v_phi_drift, 0., atol=4e-4)
-        np.testing.assert_allclose(v_phi - v_phi_drift, v_phi_ref, atol=4e-4)
+        np.testing.assert_allclose(v_phi_drift, 0., atol=4e-3)
+        np.testing.assert_allclose(v_phi - v_phi_drift, v_phi_ref, atol=4e-3)
 
 
