# Cython 3.0.4 is supported, although it generates many warnings
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -245,3 +245,3 @@
   find_package(Python 3.11 REQUIRED COMPONENTS Interpreter Development NumPy)
-  find_package(Cython 3.0.8...<3.1.0 REQUIRED)
+  find_package(Cython 3.0.4...<3.1.0 REQUIRED)
   find_program(IPYTHON_EXECUTABLE NAMES jupyter ipython3 ipython)
# skip exception tests, which can crash hdf5 depending on how it was built
--- testsuite/python/CMakeLists.txt
+++ testsuite/python/CMakeLists.txt
@@ -386,3 +386,2 @@
 python_test(FILE h5md.py MAX_NUM_PROC 2)
-python_test(FILE h5md.py MAX_NUM_PROC 1 SUFFIX 1_core)
 python_test(FILE p3m_fft.py MAX_NUM_PROC 6)
